dotnet-desktop-guide/net/winforms/whats-new/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: What's new in Windows Forms
+description: Learn about what's new in Windows Forms for .NET. Windows Forms. .NET provides new features and enhancements over .NET Framework.
+ms.date: 11/05/2020
+ms.topic: conceptual
+---
+
+# What's new (Windows Forms.NET)
+
+Windows Forms for .NET adds the following features and enhancements over .NET Framework.
```

Comment:
From .NET Fx to .NET sounds awkward, might be better to specify .NET 5 at least once or twice up front.

```suggestion
Windows Forms for .NET 5.0 adds the following features and enhancements over .NET Framework.
```